src/components/Container.js

```diff
@@ -137,40 +143,41 @@ class Container extends React.Component {
         />
       );
     }
-    // do some if to choose to render the TrailForm here instead of TrailList
+
+    if (editableTrail) {
+      return (
+        <TrailForm
+          interactionChanged={interactionChanged}
+          interaction={interaction}
+          trailEditable={trailEditable}
```

Comment:
Actions should use verb names for clarity, and to distinguish them from state properties. `trailEditable` sounds more like a property. How about something like `toggledEditing` or `enteredTrailEditing`